### ISSUE TITLE
[Flow] How to know active collectors

### ISSUE THREAD

**Issue by PatilShreyas** [Kotlin/kotlinx.coroutines#2052]
Is there any way by which we can get the number of collectors of `Flow`? 

For e.g., If I have a flow named `myFlow` and if it's collected 2 times. Then we should know number of collectors of `myFlow`.

**Comment by qwwdfsad** [comment]
It will be available in `SharedFlow` (#2034 ) out of the box.

We don't have a solution out of the box, but you can use something like this:

```
val collectors = AtomicInteger(0)
myFlow.onStart { collectors.incrementAndGet() }.onCompletion { collectors.decrementAndGet() }
```

**Comment by PatilShreyas** [comment]
@qwwdfsad okay. Thanks for this.
